### PR TITLE
Fix bugfix check (due to unknown commit_logs_cache_size_threshold)

### PR DESCRIPTION
**Pull request by azat**
Bugfix check is broken on CI [1], because of:

    2024.02.23 06:12:59.294500 [ 757 ] {} Application: Code: 115. DB::Exception: Unknown setting latest_logs_cache_size_threshold: in Coordination settings config. (UNKNOWN_SETTING), Stack trace (when copying this message, always include the lines below):

  [1]: https://s3.amazonaws.com/clickhouse-test-reports/60031/89eb2a38c0a7c113b7b0a96c8069fea6eb48ae9d/bugfix_validation.html

### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Comment by robot-clickhouse-ci-1** [comment]
*This is an automated comment for commit 83e7f3e1a1be759775776838b9e11bccc298a10c with description of existing statuses. It's updated for the latest CI running*

[⏳ Click here](https://s3.amazonaws.com/clickhouse-test-reports/60375/83e7f3e1a1be759775776838b9e11bccc298a10c/ci_running.html) to open a full report in a separate page

<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/60375/83e7f3e1a1be759775776838b9e11bccc298a10c/ci_running.html">⏳ pending</a></td></tr>
<tbody>
</table>

**Review by alexey-milovidov — APPROVED** [review]

File: docker/test/stateless/run.sh [hunk 1]
```diff
@@ -59,6 +59,16 @@ if [[ -n "$BUGFIX_VALIDATE_CHECK" ]] && [[ "$BUGFIX_VALIDATE_CHECK" -eq 1 ]]; th
...
+
+    function remove_keeper_config()
+    {
+        sudo cat /etc/clickhouse-server/config.d/keeper_port.xml \
+          | sed "/<$1>$2<\/$1>/d" \
+          > /etc/clickhouse-server/config.d/keeper_port.xml.tmp
+        sudo mv /etc/clickhouse-server/config.d/keeper_port.xml.tmp /etc/clickhouse-server/config.d/keeper_port.xml
+    }
+    # commit_logs_cache_size_threshold setting doesn't exist on some older versions
+    remove_keeper_config "commit_logs_cache_size_threshold" "[[:digit:]]\+"
```
Comment (antonio2368) [inline docker/test/stateless/run.sh:71]: `latest_logs_cache_size_threshold` also needs to be removed

Comment (antonio2368) [inline docker/test/stateless/run.sh:71]:
```suggestion
    remove_keeper_config "commit_logs_cache_size_threshold" "[[:digit:]]\+"
    remove_keeper_config "latest_logs_cache_size_threshold" "[[:digit:]]\+"
```

**Review by antonio2368 — APPROVED** [review]

**Review by alexey-milovidov — APPROVED** [review]

**Comment by alexey-milovidov** [comment]
Something with integration tests.

**Comment by azat** [comment]
This should not be related, and looks like it had been fixed in https://github.com/ClickHouse/ClickHouse/pull/60398

**Pull request merged**